routes/attribution.js

```diff
@@ -5,20 +5,34 @@ const prefix = require('./__utils__/path')('/attribution');
 
 const routes = [];
 
+const attributionMock = {
+  license: 'CC BY-SA 3.0',
+  attribution_plain:
+    'Pierre Dalous (https://commons.wikimedia.org/wiki/File:Pair_of_Merops_apiaster_feeding.jpg), "Pair of Merops apiaster feeding", https://creativecommons.org/licenses/by-sa/3.0/legalcode',
+  attribution_text:
+    'Pierre Dalous (https://commons.wikimedia.org/wiki/File:Pair_of_Merops_apiaster_feeding.jpg), "Pair of Merops apiaster feeding", https://creativecommons.org/licenses/by-sa/3.0/legalcode',
+  attribution_html:
+    'Pierre Dalous (https://commons.wikimedia.org/wiki/File:Pair_of_Merops_apiaster_feeding.jpg), "Pair of Merops apiaster feeding", https://creativecommons.org/licenses/by-sa/3.0/legalcode',
+  license_url: 'https://creativecommons.org/licenses/by-sa/3.0/legalcode',
+};
 routes.push({
-  path: prefix('/{file}/and-so-on'),
+  path: prefix('/{language}/{file}'),
```

Comment:
This should be fine for the mock endpoint!
As we already discussed though, we should revisit the parameter structure later on.
Having `'/'`-separated parameters with optional parameters in between (`language_code`) will introduce some ambiguities that make parameter handling unnecessarily complex.